docs/getting-started/introduction.md

```diff
@@ -1,121 +1,223 @@
 
-::: tip What's in a name?
+<!--
+@TODO getting-started > learn-directus
+For more information on Directus-specific terms, see the guide on [how to learn Directus](/getting-started/learn-directus).
+-->
 
-"Directus" ([duh REKT iss](http://audio.pronouncekiwi.com/Salli/Directus)) is Latin for: _laid straight, arranged in
-lines_. The broadest goal of our platform is to present data in a simple, orderly, and intuitive way.
+:::
+
+<!--
+:::tip Ready to see what Directus can do?
+
+Checkout the core features and learn Directus hands-on in the [Quickstart Guide](/getting-started/quickstart/).
+
+@TODO getting-started > crashcourse
+Change this CTA to the 20-30 min crashcourse
+
+:::
+-->
+
+## When to Use It
+
+**Directus can power any data-driven project and can be linked or removed at any time.**
+
+You'll be able to build, monitor or manage any data model or app you desire: IoT fleets, ecommerce, SaaS, business data
+analytics, multi-channel content, _or anything else_.... If it involves SQL and digital files, you can use Directus.
+With this in mind, four broad use cases emerge:
+
+- **Backend as a Service**\
+  An end-to-end data solution. Efficiently connects data, auto-generates exhaustive APIs for projects that scale, provides
+  webhooks and task automation. Completely detached from the database, you're free to link and remove Directus anytime or
+  use any other service in tandem.
+- **Headless CMS**\
+  Remember: _content_ is just data from a database! Manage any omnichannel digital experience. Deliver file assets and data
+  across websites, apps, kiosks, digital signage... _the sky's the limit!_
+- **Internal Tool Builder**\
+  The whole team can build custom apps! Ditch the spreadsheet and quickly build back-office apps and admin panels for customers,
+  inventory, projects, marketing _or anything else._
+- **Data Management and Analytics**\
+  Establish _a single source of truth_ for all data. Build no-code analytics dashboards to gain insights into company KPIs
+  and other metrics. Coalesce previously siloed department data.
+
+<!--
+mention SaaS in there
+
+:::tip Ready to dive-in?
+
+Get a Project running in minutes. Learn Directus hands-on in the [Quickstart Guide](/getting-started/quickstart/).
+
+:::
+-->
+
+## Why Use It?
+
+**Directus is a simple, data-first solution to complex problems and using it has no downsides.**
+
+Directus was created in 2004, and has been slowly, iteratively improved on for a long time. The core team has carefully
+thought through the app, beginning-to-end, to make sure that using this platform is an all-benefit and no downside
+experience. This platform is plug-and-play, so you're free to link or remove it any time, with no impact on your data.
+You have no vendor lock-in whatsoever, your data is yours. There are no artificial limits on data _(e.g. limits on users
+or roles)_.
+
+<!-- highlight the cost-of-scaling that comes with a platform which charges for users and roles -->
```

Comment:
@benhaynes Had this idea too, you want me to hit the trigger and add in that argument more explicitly here?